src/measure_theory/integral/set_to_l1.lean

```diff
@@ -567,6 +576,60 @@ begin
   exact set_to_L1s_indicator_const hT hs hμs x,
 end
 
+lemma norm_set_to_L1_le_norm_set_to_L1s_clm (hT : dominated_fin_meas_additive μ T C) :
+  ∥set_to_L1 hT∥ ≤ ∥set_to_L1s_clm α E μ hT∥ :=
+calc ∥set_to_L1 hT∥
+    ≤ (1 : ℝ≥0) * ∥set_to_L1s_clm α E μ hT∥ : begin
+      refine continuous_linear_map.op_norm_extend_le (set_to_L1s_clm α E μ hT) (coe_to_Lp α E ℝ)
+        (simple_func.dense_range one_ne_top) (λ x, le_of_eq _),
+      rw [nnreal.coe_one, one_mul],
+      refl,
+    end
+... = ∥set_to_L1s_clm α E μ hT∥ : by rw [nnreal.coe_one, one_mul]
+
+lemma norm_set_to_L1_le_mul_norm (hT : dominated_fin_meas_additive μ T C) (hC : 0 ≤ C)
+  (f : α →₁[μ] E) :
+  ∥set_to_L1 hT f∥ ≤ C * ∥f∥ :=
+calc ∥set_to_L1 hT f∥
+    ≤ ∥set_to_L1s_clm α E μ hT∥ * ∥f∥ :
+  continuous_linear_map.le_of_op_norm_le _ (norm_set_to_L1_le_norm_set_to_L1s_clm hT) _
+... ≤ C * ∥f∥ : mul_le_mul (norm_set_to_L1s_clm_le hT hC) le_rfl (norm_nonneg _) hC
+
+lemma norm_set_to_L1_le_mul_norm' (hT : dominated_fin_meas_additive μ T C) (f : α →₁[μ] E) :
+  ∥set_to_L1 hT f∥ ≤ max C 0 * ∥f∥ :=
+calc ∥set_to_L1 hT f∥
+    ≤ ∥set_to_L1s_clm α E μ hT∥ * ∥f∥ :
+  continuous_linear_map.le_of_op_norm_le _ (norm_set_to_L1_le_norm_set_to_L1s_clm hT) _
+... ≤ max C 0 * ∥f∥ :
+  mul_le_mul (norm_set_to_L1s_clm_le' hT) le_rfl (norm_nonneg _) (le_max_right _ _)
+
+lemma norm_set_to_L1_le (hT : dominated_fin_meas_additive μ T C) (hC : 0 ≤ C) :
+  ∥set_to_L1 hT∥ ≤ C :=
+continuous_linear_map.op_norm_le_bound _ hC (norm_set_to_L1_le_mul_norm hT hC)
+
+lemma norm_set_to_L1_le' (hT : dominated_fin_meas_additive μ T C) :
+  ∥set_to_L1 hT∥ ≤ max C 0 :=
+continuous_linear_map.op_norm_le_bound _ (le_max_right _ _) (norm_set_to_L1_le_mul_norm' hT)
+
+lemma set_to_L1_lipschitz (hT : dominated_fin_meas_additive μ T C) :
+  lipschitz_with (real.to_nnreal C) (L1.set_to_L1 hT) :=
+begin
+  have h_lip : lipschitz_with (∥L1.set_to_L1 hT∥₊) (L1.set_to_L1 hT),
+    from continuous_linear_map.lipschitz _,
+  -- TODO: is there some lipschitz_with.mono lemma?
```

Comment:
`lipschitz_with.weaken`? (Should probably be renamed `lipschitz_with.mono`, indeed).

Reply:
Thanks!